.editorconfig

```diff
@@ -14,6 +14,21 @@ x-soft-wrap-limit = 120
 x-show-invisibles = false
 x-show-spaces = false
 
+[*.{java}]
+end_of_line = lf
+insert_final_newline = true
+indent_style = tab
```

Comment:
This doesn't seem compatible to how the Java files are styled. @Mause I can revert this in a PR I'm working on.